### ISSUE TITLE
Priorities for different tasks

### ISSUE THREAD

**Issue by tejuamirthi** [IEEE-VIT/ToDo-iOS#39]
**Is your feature request related to a problem? Please describe.**
Tasks are stored and we don't have any indication for their priority. So cannot filter high priority tasks.

**Describe the solution you'd like**
**High/Medium/Low** priorities to be added for tasks. This need to be display to the right of the task name as a tag, just like the labels we see for different issues.

**Comment by tejuamirthi** [comment]
@aaryankotharii Do mark this as duplicate for #38 if you feel status and priority can be taken up together!

**Comment by aaryankotharii** [comment]
This is something that can be done since we will be sticking to only 3 test cases. i.e low, medium, high. An enum for priority  The following SF symbols shall be used for indicating priority :
<img src = "https://user-images.githubusercontent.com/53724307/94951083-4e32f280-0501-11eb-93d0-ea0986b98cfd.jpeg" width = "350" >

Here is a sample UI for this feature:
<img src = "https://user-images.githubusercontent.com/53724307/94951340-bb468800-0501-11eb-87e8-73f3b1d514cb.png" width = "350" >

**Comment by aaryankotharii** [comment]
@tejuamirthi would you like to work on this issue ?

**Comment by tejuamirthi** [comment]
yes @aaryankotharii . would like to implement the same with an enum

**Comment by aaryankotharii** [comment]
Assigned.
